pkg/controller/gitopsconfig/controller.go

```diff
@@ -427,11 +427,11 @@ func (r *Reconciler) manageDeletion(instance *gitopsv1alpha1.GitOpsConfig) (reco
 		}
 		return reconcile.Result{}, nil
 	}
-	//if it's not succeeded we wait for 1 minute
+	//if it's not succeeded we wait for half a minute
 	//TODO add logic to stop at a certain point ... or not ...
 	return reconcile.Result{
 		Requeue:      true,
-		RequeueAfter: time.Minute,
+		RequeueAfter: time.Second * 30,
```

Comment:
As we discussed, I would suggest one time amount here and in l.417, and removing the reference to jobmonitor.go (here and in commit message), and adding a TODO comment about exponential backoff, maybe like in CronJob, if you believe it makes sense.